src/components/action-popover/__snapshots__/action-popover.spec.js.snap

```diff
@@ -5109,7 +5109,7 @@ exports[`ActionPopover submenu right aligned opens the submenu when right key is
                   "displayName": "ActionPopoverItem",
                   "foldedComponentIds": Array [],
                   "render": [Function],
-                  "styledComponentId": "sc-dnqmqq",
+                  "styledComponentId": "sc-gqjmRU",
```

Comment:
looks like we have really bad snapshots here, could you change the renderer in the spec file to fix that?

Reply:
I think making a snapshot of wrapper rendered by enzyme's `shallow` will greatly reduce it

Reply:
Good spot I'll see what I can do 💪

Reply:
I ended up using `TestRenderer` as I need to go deeper into the render tree